engineapi/engineapi/models.py

```diff
@@ -346,6 +347,11 @@ class Leaderboard(Base):  # type: ignore
     title = Column(VARCHAR(128), nullable=False)
     description = Column(String, nullable=True)
     resource_id = Column(UUID(as_uuid=True), nullable=True, index=True)
+    blockchain_ids = Column(ARRAY(Integer), nullable=False, default=[])
```

Comment:
I sugest also add `sort` for add and update values to this list, so it always will be readable, for example when query returns few result:
```
    id               blockchain_ids
<uuid1>             80001, 1
<uuid2>             1, 80001
```